File: Ejercicios Resueltos/Lenguaje C/Ejercicios Basicos/Ejercicio-28/ejercicio28.c
// Ejercicio 28
// Hacer un programa que ingrese por teclado un número total de segundos y que luego
// pueda mostrar la cantidad de horas, minutos y dias que existen en el valor
// ingresado.

#include <stdio.h>

void main(){
    
    int segundos, minutos, horas, dias;

    printf("Ingrese la cantidad de segundos: ");
    scanf("%d",&segundos);

    minutos=segundos/60;
    horas=minutos/60;
    dias=horas/24;

    printf("%d segundos equivalen a %d minutos, %d horas, %d dias.\n",segundos,minutos,horas,dias);

    printf("Presione ENTER para continuar...");
    getchar();
    getchar();

}

File: Ejercicios Resueltos/Lenguaje Python/Ejercicios Basicos/Ejercicio-28/ejercicio28.py
# Ejercicio 28
# Hacer un programa que ingrese por teclado un número total de segundos y que luego
# pueda mostrar la cantidad de horas, minutos y segundos que existen en el valor
# ingresado.

print("EJERCICIO 28")

segundos = int(input("Ingrese la cantidad de segundos: "))
horas = int(segundos/3600)
segundosRestantes = segundos%3600
minutos = int(segundosRestantes/60)
segundosRestantes = int(segundosRestantes%60)

print(f'\n{segundos} segundos equivales a {horas} horas, {minutos} minutos con {segundosRestantes} segundos')

input("\n\nPresione ENTER para continuar...")